spec/models/project_spec.rb

```diff
@@ -11,4 +11,38 @@
     it {should have_many :contestant_projects}
     it {should have_many(:contestants).through(:contestant_projects)}
   end
+
+  describe "#avg_exp" do
+    it "returns the average years of experience (rounded to one dec) of all contestants on a project" do
+      @recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+      @furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+  
+      @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+      @boardfit = @recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+  
+      @upholstery_tux = @furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+      @lit_fit = @furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+  
+      @jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+      @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+      @kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+      @erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+  
+  
+      ContestantProject.create(contestant_id: @jay.id, project_id: @news_chic.id)
+      ContestantProject.create(contestant_id: @gretchen.id, project_id: @news_chic.id)
+      ContestantProject.create(contestant_id: @gretchen.id, project_id: @upholstery_tux.id)
+      ContestantProject.create(contestant_id: @kentaro.id, project_id: @upholstery_tux.id)
+      ContestantProject.create(contestant_id: @kentaro.id, project_id: @boardfit.id)
+      ContestantProject.create(contestant_id: @erin.id, project_id: @boardfit.id)
+
+      expect(@news_chic.avg_exp).to eq(12.5)
+
+      expect(@boardfit.avg_exp).to eq(11.5)      
+
+      expect(@upholstery_tux.avg_exp).to eq(10)      
+
+      expect(@lit_fit.avg_exp).to eq(0)
+    end
+  end
```

Comment:
Great work testing this logic at the model level and the feature level! When encapsulating the contestant count logic at the model level as well, you'd want to test for that too